test/functional/test_framework/test_framework.py

```diff
@@ -391,9 +391,13 @@ def setup_nodes(self):
                 assert_equal(chain_info["initialblockdownload"], False)
 
     def import_deterministic_coinbase_privkeys(self):
-        wallet_names = [self.default_wallet_name] * len(self.nodes) if self.wallet_names is None else self.wallet_names
-        assert len(wallet_names) <= len(self.nodes)
```

Comment:
Could keep the assert as

```
assert len(self.wallet_names) <= len(self.nodes)

Reply:
re: https://github.com/bitcoin/bitcoin/pull/20186#discussion_r507723131

> Could keep the assert as
> 
> ```
> assert len(self.wallet_names) <= len(self.nodes)
> ```

Added back